### PR TITLE
GothamAds Bid Adapter: add at, ccpa, gdpr and coppa support 

### PR DESCRIPTION
**Pull request by supportGothamad**
## Type of change

- [ ] Bugfix
- [ x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

update gothamAds adapter: add at, ccpa, gdpr and coppa support 

- test parameters for validating bids
```
    {
                bidder: 'gothamads',
                params: {
                        placementId: 'hash',
                    accountId: 'accountId'
                }
            }
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by patmmccann** [comment]
Please see also #6466

**Comment by ChrisHuie** [comment]
@supportGothamad can you add docs updates to go along with this?

File: modules/gothamadsBidAdapter.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+import {
+  BANNER,
+  NATIVE,
+  VIDEO
+} from '../src/mediaTypes.js';
 import * as utils from '../src/utils.js';
-import {config} from '../src/config.js';
+import {
+  config
+} from '../src/config.js';
```
Comment (FilipStamenkovic) [inline modules/gothamadsBidAdapter.js:12]: This is personal preference so you don't have to change it if you disagree, but I think previously imports were more readable (when import was in single line, not in multiple lines).

File: modules/gothamadsBidAdapter.js [hunk 2]
```diff
@@ -94,9 +109,23 @@ export const spec = {
...
           tid: bidRequest.transactionId
         },
+        regs: {
+          coppa: config.getConfig('coppa') === true ? 1 : 0,
+          ext: {}
+        },
         tmax: bidRequest.timeout,
         imp: [impObject],
       };
+      if (bidRequest) {
```
Comment (FilipStamenkovic) [inline modules/gothamadsBidAdapter.js:119]: You don't actually need this `if`.  
In couple of lines above you are already retrieving properties from `bidRequest` object (for example line 116: `bidRequest.timeout`).  
So, in this line `bidRequest` would **always** evaluate to true.

**Review by FilipStamenkovic — CHANGES_REQUESTED** [review]
Code looks ok, but you are missing docs PR, like @ChrisHuie already mentioned.  
Also, please take a look at @patmmccann comment.

File: modules/gothamadsBidAdapter.js (hunk 2 above)
Comment (supportGothamad) [reply modules/gothamadsBidAdapter.js]: @FilipStamenkovic  Hello! Thank you! I have fixed adapter according to your comments!

**Comment by supportGothamad** [comment]
@ChrisHuie Hello! I have update docs also

**Review by FilipStamenkovic — APPROVED** [review]
LGTM. Can you just paste link of docs PR? I can't find it.

**Comment by ChrisHuie** [comment]
@supportGothamad just following up that a docs pr was created for this and submitted to our docs repo so we can merge this -> https://github.com/prebid/prebid.github.io

**Pull request merged**